spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -452,85 +451,10 @@ abstract class Graph[T: ClassTag](
   }
 }
 
-object Graph extends ContainerSerializable {
-  /**
-   * Node for graph container. The module should have a tensor/table input while a tensor output
-   * @tparam T
-   */
-  type ModuleNode[T] = Node[AbstractModule[Activity, Activity, T]]
-
-  /**
-   * Build multiple inputs, multiple outputs graph container.
-   * @param input input node
-   * @param output output node
-   * @return a graph container
-   */
-  def apply[T: ClassTag](
-    input : Array[ModuleNode[T]],
-    output : Array[ModuleNode[T]],
-    variables: Option[(Array[Tensor[T]], Array[Tensor[T]])] = None
-  )(implicit ev: TensorNumeric[T]) : Graph[T] = {
```

Comment:
@yiheng  no functional changes. just extract a method to make it extendable by the subclass.